BenchBnb/BenchBnb/Views/Bench/Create.cshtml

```diff
@@ -5,7 +5,7 @@
 
 <h2>Index</h2>
```

Comment:
Make sure that your page headings contain appropriate text.

Reply:
Fixed